src/Messenger/IndexationRequest.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+namespace JoliCode\Elastically\Messenger;
+
+final class IndexationRequest
+{
+    private $op;
```

Comment:
don't be lazy :D 
Could you rename it to `operation` (and everywhere else?) It will enchance the DX

Reply:
Named it that way because the property is serialized and stored in a bus. So `operation` is 7 bytes heavier... I will rename it for the sake of DX :+1: